docs/diagrams/CommandClasses.puml

```diff
@@ -0,0 +1,107 @@
+@startuml
+'!include style.puml
+'skinparam arrowThickness 1.1
+'skinparam arrowColor UI_COLOR_T4
+'skinparam classBackgroundColor UI_COLOR
```

Comment:
Perhaps you could change the color to match the boba theme (similar to the activity diagrams) 🧋